website/docs/docs/dbt-versions/release-notes/09-Mar-2023/public-preview-trino-in-dbt-cloud.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "New: Starbust integration with public preview of dbt-trino in dbt Cloud"
+id: "public-preview-trino-in-dbt-cloud"
+description: "Public preview of dbt Cloud integrations with Starburst and Trino clusters is now available."
+sidebar_label: "New: Starbust integration with public preview of dbt-trino in dbt Cloud"
+tags: [March-30-2023]
```

Comment:
is there another tag we should add to the current ^ list? wdyt about `adapter` or `connection` ? something else?